src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -188,7 +188,8 @@ protected void scrollAndClick(WebElement element, int offset) {
   protected void scrollToElement(WebElement element) {
     try {
       ((JavascriptExecutor) driver).executeScript("var x = $(arguments[0]);"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "var x = $(arguments[0]);" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)